test/robustness/overlay/buffer/recursive_polygons_buffer.cpp

```diff
@@ -133,27 +129,34 @@ bool verify(std::string const& caseid, MultiPolygon const& mp, MultiPolygon cons
         wkt = true;
     }
 
-    if (svg)
+    std::string filename;
+
     {
-        std::ostringstream filename;
-        filename << caseid << "_"
-            << string_from_type<typename bg::coordinate_type<MultiPolygon>::type>::name()
-#if defined(BOOST_GEOMETRY_USE_RESCALING)
+        // Generate a unique name
+        std::ostringstream out;
+        out << "rec_pol_buffer_" << geometry_to_crc(mp)
+            << "_" << string_from_type<typename bg::coordinate_type<MultiPolygon>::type>::name()
+       #if defined(BOOST_GEOMETRY_USE_RESCALING)
             << "_rescaled"
-#endif
-            << ".svg";
-        create_svg(filename.str(), mp, buffer);
+       #endif
+            << ".";
+        filename = out.str();
+    }
+
+    if (svg)
+    {
+        create_svg(filename + "svg", mp, buffer);
     }
 
     if (wkt)
     {
-        std::ostringstream filename;
-        filename << caseid << "_"
-            << typeid(typename bg::coordinate_type<MultiPolygon>::type).name()
-            << ".wkt";
-        std::ofstream stream(filename.str().c_str());
+        std::ofstream stream(filename + "wkt");
         stream << bg::wkt(mp) << std::endl;
-        stream << bg::wkt(buffer) << std::endl;
+        if (false)
```

Comment:
is this part of code needed?

Reply:
Not really, I can omit it. If it is wished it's trivial to add again.